codespell_lib/data/dictionary.txt

```diff
@@ -6823,12 +6882,34 @@ declaratinos->declarations
 declaratiom->declaration
 declaraton->declaration
 declaratons->declarations
+declard->declared
+declarded->declared
 declaritive->declarative
 declaritively->declaratively
+declarnig->declaring
+declars->declared, declares,
+declartated->declarated
```

Comment:
I wonder if this should also suggest declared?

Reply:
Also Travis says this word is also a typo:
error declartated: correction declarated is an error itself